test/versioned/aws-sdk-v3/bedrock-chat-completions.tap.js

```diff
@@ -98,7 +102,8 @@ tap.afterEach(async (t) => {
   { modelId: 'anthropic.claude-v2', resKey: 'claude' },
   { modelId: 'anthropic.claude-3-haiku-20240307-v1:0', resKey: 'claude3' },
   { modelId: 'cohere.command-text-v14', resKey: 'cohere' },
-  { modelId: 'meta.llama2-13b-chat-v1', resKey: 'llama2' }
+  { modelId: 'meta.llama2-13b-chat-v1', resKey: 'llama2' },
+  { modelId: 'meta.llama3-8b-instruct-v1:0', resKey: 'llama3' }
```

Comment:
based on comments above you could just use the resKey of `llama2` or make that more generic that way we reduce the amount of mock code to maintain